srcDLL/op2ext.cpp

```diff
@@ -99,6 +99,12 @@ OP2EXT_API void Log(const char* message)
 	logger.Log(message, FindModuleName(_ReturnAddress()));
 }
 
+
+OP2EXT_API bool IsModuleLoaded(const char* moduleName)
+{
+	return IsConsoleModuleLoaded(moduleName) || IsIniModuleLoaded(moduleName);
```

Comment:
For consistency with the method in the later commit, perhaps `IsIniModuleLoaded` should be checked first. It may be good to always checked the two module systems in the same order. In this case I know it really doesn't matter though. It might even make slightly more sense from an efficiency point to check in the order given here, as expected usage is probably with console modules.

I'm going to say no change required.